handlers/issue.go

```diff
@@ -19,53 +18,58 @@ func (s *Server) handleIssueEvent(body []byte) {
 
 }
 
-func (s *Server) handleIssueCommentEvent(body []byte) {
-	glog.Infof("Received an IssueComment Event")
-
+//function to handle issue comments
+func (s *Server) handleIssueCommentEvent(body []byte, client *github.Client) {
 	var prc github.IssueCommentEvent
+
 	err := json.Unmarshal(body, &prc)
 	if err != nil {
 		glog.Errorf("fail to unmarshal: %v", err)
 	}
 	glog.Infof("prc: %v", prc)
+
+	owner := *prc.Repo.Owner.Login
```

Comment:
Do we need to login every time we handle issue comment? Can't we login once and use the context everywhere?

Reply:
@m1093782566 , here we not logging in every time, we are just taking the owner info from the payload itself, will be using this owner info to get the collaborator info from the go-GitHub api

Reply:
It's a bit confusing though. Please comment there.